sim/rogue/combat/revealing_strike.go

```diff
@@ -0,0 +1,81 @@
+package combat
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+	"github.com/wowsims/cata/sim/rogue"
+)
+
+func (comRogue *CombatRogue) registerRevealingStrike() {
+	if !comRogue.Talents.RevealingStrike {
+		return
+	}
+
+	hasGlyph := comRogue.HasPrimeGlyph(proto.RoguePrimeGlyph_GlyphOfRevealingStrike)
+	actionID := core.ActionID{SpellID: 84617}
+
+	// Enemy Debuff Aura for Finisher Damage
+	rvsAura := comRogue.NewEnemyAuraArray(func(target *core.Unit) *core.Aura {
+		return target.GetOrRegisterAura(core.Aura{
+			Label:    "Revealing Strike",
+			ActionID: actionID,
+			Duration: 15 * time.Second,
+
+			// Technically this _could_ cause problems in a target swapping situation, but it's good enough.
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				comRogue.Eviscerate.DamageMultiplierAdditive += core.TernaryFloat64(hasGlyph, .45, .35)
```

Comment:
you an set the value once in the register instead of calling TernaryFloat so many times when its all the same and not dynamic